### ISSUE TITLE
Softmax warning using PyTorch backend on Mac M1

### ISSUE THREAD

**Issue by etjones** [usefulsensors/moonshine#16]
Running a transcription on my 2020 Macbook Air M1 with `KERAS_BACKEND=torch`, the following warning is printed to stderr, although transcription proceeds flawlessly (Thanks!)

```shell
/Users/<USER>/Desktop/moonshine_live/.venv/lib/python3.11/site-packages/keras/src/ops/nn.py:545: 
UserWarning: You are using a softmax over axis 3 of a tensor of shape 
torch.Size([1, 8, 1, 1]). This axis has size 1. The softmax operation will always 
return the value 1, which is likely not what you intended. 
Did you mean to use a sigmoid instead?
```

**Comment by keveman** [comment]
For the first token in the decoding loop, the attention score matrix's shape is 1x1, and that triggers the warning you point to. It's benign and you can ignore it. Or run with `python -W ignore` to make python not print them out. Closing, as there are no actionable followups.

**Comment by damascene** [comment]
I suggest adding this to the documentation as people will keep searching and finding this issue and reporting it.
